.github/workflows/environment.yml

```diff
@@ -18,4 +19,3 @@ jobs:
       - run: docker login https://docker.pkg.github.com -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
       - uses: actions/checkout@v2
       - run: make environment-prepare
-      - run: docker push docker.pkg.github.com/timberio/vector/environment:${{ github.sha }}
```

Comment:
Randomly spot this, and wondering is this correct?
Maybe an `if: github.event_name != 'workflow_dispatch'` would be a better fit?